src/components/App/theme.css

```diff
@@ -165,3 +165,7 @@ a {
   text-decoration: none;
   color: var(--ion-color-modus-charcoal);
 }
+
+*:focus {
+  outline: transparent;
```

Comment:
a11y killer

Reply:
We want to modify this for this app?

Reply:
I kinda believe in a11y. I mean, you could be having the Sun at your screen, right? That's also accessibility

Reply:
ok.  fixed